database/postgresql/src/main/resources/application.properties

```diff
@@ -3,3 +3,4 @@ quarkus.datasource.username=${QUARKUS_DATASOURCE_USERNAME}
 quarkus.datasource.password=${QUARKUS_DATASOURCE_PASSWORD}
 quarkus.datasource.jdbc.url=jdbc:postgresql://localhost:5432/main_postgres_db
 quarkus.hibernate-orm.physical-naming-strategy=org.hibernate.boot.model.naming.CamelCaseToUnderscoresNamingStrategy
+quarkus.hibernate-orm.log.sql=true
```

Comment:
Let's add this to -dev or -local, or other profiles. The 'default'/production should not log SQLs.

Reply:
changed!